### DISCUSSION TITLE
Info variazione Mqtt da versione 2024.2.0

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2573] [GENERALE - Discussioni generiche]
ciao a tutti, chiedo una delucidazione.
mi compare il warning che dalla versione 2024.2.0 ci son variazioni sui nomi/disposizioni Mqtt.

nel mio caso:
----
Alcune entità MQTT hanno un nome di entità che inizia con il nome del dispositivo. Questo non è previsto. Per evitare un nome duplicato, il prefisso del nome del dispositivo viene privato del nome dell'entità come soluzione alternativa. Si prega di informare il manutentore dell'applicazione software che fornisce le entità interessate per risolvere questo problema.

Elenco delle entità interessate:

sensor.temp_qualita_soggiorno_formaldehyd
sensor.temp_qualita_camera_temperature
sensor.temp_bagno_pressure
sensor.temp_bagno_temperature
sensor.temp_camera_grande_temperature
sensor.temp_loc_tecnico_humidity
sensor.temp_camera_grande_humidity

prendiamo come esempio:
sensor.temp_camera_grande_temperature 
- entità: sensor.temp_camera_grande_temperature   
- nome entità: Temp_Camera_Grande _temperature  
- nome dispositivo: Temp_Camera_Grande
in pratica l'entità non potrà più iniziare con il nome del dispositivo giusto? 
considerato lo sbatti di cambiare entità, meglio cambiare il nome dispositivo e lasciare l'entità invariata?

grazie in anticipo.

**Comment by indomus** [comment]
https://github.com/indomus/forum/discussions/2568
